pytorch_lightning/callbacks/pt_callbacks.py

```diff
@@ -254,6 +255,30 @@ def on_epoch_end(self, epoch, logs=None):
                 self.save_model(filepath, overwrite=False)
 
 
+class GradientAccumulationScheduler(Callback):
+    """Change gradient accumulation factor according to scheduling.
+    # Arguments
+        scheduling: dict, scheduling in format {epoch: accumulation_factor}
+    """
+    def __init__(self, scheduling: dict):
+        minimal_epoch = min(scheduling.keys())
```

Comment:
this will fails with an empty dictionary
```
min(dict().keys())
Traceback (most recent call last):
  File "/home/jb/.local/lib/python3.6/site-packages/IPython/core/interactiveshell.py", line 3265, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
  File "<ipython-input-2-763e8ea42ef8>", line 1, in <module>
    min(dict().keys())
ValueError: min() arg is an empty sequence
```

Reply:
Added empty dict error